contracts/Application.sol

```diff
@@ -3,15 +3,15 @@
 pragma solidity ^0.8.17;
 
 abstract contract Application {
-    address public immutable MESSAGE_PORT;
+    address public immutable LINE_REGISTRY;
 
-    constructor(address msgPort) {
-        MESSAGE_PORT = msgPort;
+    constructor(address lineRegistry) {
+        LINE_REGISTRY = lineRegistry;
```

Comment:
LineRegistry would not call `Application`.
source: app -> line -> low message protocol
target: low message protocol -> line -> app